core/tests/protractor_utils/AdminPage.js

```diff
@@ -154,6 +154,72 @@ var AdminPage = function() {
     await waitFor.pageToFullyLoad();
   };
 
+  this.getFeaturesTab = async function() {
```

Comment:
Should this maybe be ``openFeaturesTab``? It doesn't seem to actually be getting/returning anything.